pandas-stubs/core/frame.pyi

```diff
@@ -1776,7 +1776,7 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def rolling(
         self,
-        window: int | BaseOffset | BaseIndexer,
+        window: int | str | BaseOffset | BaseIndexer,
```

Comment:
changes to `core/frame.pyi` should be in a different PR

Reply:
Fixed by rebase.